algorithms/src/snark/marlin/ahp/ahp.rs

```diff
@@ -110,33 +108,22 @@ impl<F: PrimeField, MM: MarlinMode> AHPForR1CS<F, MM> {
     }
 
     /// Get all the strict degree bounds enforced in the AHP.
-    pub fn get_degree_bounds(info: &CircuitInfo) -> [usize; 4] {
+    pub fn get_degree_bounds(info: &CircuitInfo) -> [usize; 5] {
         let num_constraints = info.num_constraints;
+        let num_variables = info.num_variables;
         let num_non_zero_a = info.num_non_zero_a;
         let num_non_zero_b = info.num_non_zero_b;
         let num_non_zero_c = info.num_non_zero_c;
         [
             EvaluationDomain::<F>::compute_size_of_domain(num_constraints).unwrap() - 2,
```

Comment:
Do we ever perform a sumcheck over the constraint domain? I don't think so, right? We only need degree bounds for domains over which we perform sumchecks.

Reply:
Great catch: 214a406cd. This will likely be added again for lookups.